src/Tribe/Main.php

```diff
@@ -286,6 +286,23 @@ public function load_assets() {
 			]
 		);
 
+		tribe_asset(
+			$this,
+			'tribe-admin-image-field',
+			'admin-image-field.js',
+			[ 'jquery' ],
+			null,
```

Comment:
If the conditional is being called too early, make this particular JS load later in the page like the footer o the page so you can use the style of conditional that I mentioned to you on our call.

Reply:
Ah.... I realized that I was trying to use `wp_footer` and I should have been using `admin_footer`. Got it fixed. Thanks for the push!